CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 - Device sent bytes pie chart now showing up on Chrome browser.
 - Fix typos in the Trigger Editor
+- Change `/resource/:id` routes to `/resource/:id/edit` to avoid conflicts with `/resource/new` routes
```

Comment:
I'd put this under `### Changed` rather than `### Fixed` since that's where I'd expect to see changes